.gitattributes

```
@@ -0,0 +1,2 @@
+* text=auto
+*.sh text eol=lf
```

Please add mpty line at the end of your file.

```suggestion
*.sh text eol=lf

```